### ISSUE TITLE
Add License to Repository

### ISSUE THREAD

**Issue by eseglem** [wmgeolab/geoBoundaryBot#14]
Based on https://github.com/wmgeolab/geoBoundaries/blob/main/LICENSE I would guess the intent is for this repo to also be CC BY 4.0. If that is the case, it should also be included here. Otherwise, the applicable license should be added.

A 'contributing' file might also be a good idea. That is much less of a hang up than the license though.

**Comment by DanRunfola** [comment]
This is all correct; I'll try to get this added in the next week or so.  Just to remind myself, this is also an issue for the gbWeb repository, so I need to confirm we have that updated correctly as well.
